tests/base/test_logging.py

```diff
@@ -1,34 +1,42 @@
 import pytest
 
 import pydp as pd
-
+# Creating a Class TestLogging
 class TestLogging():
+    #Creating First testcase
     def test_logging(self):
         directory = 'tests'
         vlog_level = 3
         l = pd.logging.init_logging(directory, 'log_test', vlog_level)
+        #if both directory and vlog_level contains value the test will be pass
```

Comment:
This comment is misleading. 
`pd.logging.init_logging`  returns true if it's successful in initializing logs. Check for DP code: https://github.com/google/differential-privacy/blob/master/differential_privacy/base/logging.h#L289 for more.